lib/SIL/IR/AbstractionPattern.cpp

```diff
@@ -795,6 +852,9 @@ void AbstractionPattern::print(raw_ostream &out) const {
   case Kind::CXXMethodType:
   case Kind::CurriedCXXMethodType:
   case Kind::PartialCurriedCXXMethodType:
+  case Kind::CXXOperatorMethodType:
+  case Kind::CurriedCXXOperatorMethodType:
+  case Kind::PartialCurriedCXXOperatorMethodType:
     out << (getKind() == Kind::CXXMethodType
```

Comment:
Could you update the pretty-printing code?

Reply:
Done